new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleValidation.java

```diff
@@ -0,0 +1,341 @@
+    String pvcName = domainUid + "-pvc"; // name of the persistent volume claim
+    String pvName = domainUid + "-pv"; // name of the persistent volume
+
+    V1PersistentVolumeClaim v1pvc = new V1PersistentVolumeClaim()
+        .spec(new V1PersistentVolumeClaimSpec()
+            .addAccessModesItem("ReadWriteMany")
+            .storageClassName(domainUid + "-weblogic-domain-storage-class")
+            .volumeName(pvName)
+            .resources(new V1ResourceRequirements()
+                .putRequestsItem("storage", Quantity.fromString("10Gi"))))
+        .metadata(new V1ObjectMetaBuilder()
+            .withName(pvcName)
+            .withNamespace(domainNamespace1)
+            .build()
+            .putLabelsItem("weblogic.resourceVersion", "domain-v2")
+            .putLabelsItem("weblogic.domainUid", domainUid));
+
+    boolean success = assertDoesNotThrow(
+        () -> TestActions.createPersistentVolumeClaim(v1pvc),
+        "Persistent volume claim creation failed, "
+        + "look at the above console log messages for failure reason in ApiException responsebody"
+    );
+    assertTrue(success, "PersistentVolumeClaim creation failed");
+
+    V1PersistentVolume v1pv = new V1PersistentVolume()
+        .spec(new V1PersistentVolumeSpec()
+            .addAccessModesItem("ReadWriteMany")
+            .storageClassName(domainUid + "-weblogic-domain-storage-class")
+            .volumeMode("Filesystem")
+            .putCapacityItem("storage", Quantity.fromString("10Gi"))
+            .persistentVolumeReclaimPolicy("Recycle")
+            .hostPath(new V1HostPathVolumeSource()
+                .path(TestConstants.PV_ROOT + "/" + domainUid + "-persistentVolume")))
+        .metadata(new V1ObjectMetaBuilder()
+            .withName(pvName)
+            .withNamespace(domainNamespace1)
+            .build()
+            .putLabelsItem("weblogic.resourceVersion", "domain-v2")
+            .putLabelsItem("weblogic.domainUid", domainUid));
+    success = assertDoesNotThrow(
+        () -> TestActions.createPersistentVolume(v1pv),
+        "Persistent volume creation failed, "
+        + "look at the above console log messages for failure reason in ApiException responsebody"
+    );
+    assertTrue(success, "PersistentVolume creation failed");
+  }
+
+  @Test
+  @Order(1)
+  @DisplayName("Create a MII image")
+  public void testCreatingMiiImage() {
+
+    logger.info("WDT model directory is {0}", MODEL_DIR);
+
+    // build the model file list
+    final List<String> modelList = Collections.singletonList(MODEL_DIR + "/" + WDT_MODEL_FILE);
+
+    // build an application archive using what is in resources/apps/APP_NAME
+    boolean archiveBuilt = buildAppArchive(
+        defaultAppParams()
+            .srcDir(APP_NAME));
+
+    assertThat(archiveBuilt)
+        .as("Create an app archive")
+        .withFailMessage("Failed to create app archive for " + APP_NAME)
+        .isTrue();
+
+    // build the archive list
```

Comment:
in general, all of these comments should be log messages, so we can look at the output and easily understand what it is doing